### DISCUSSION TITLE
Supabase

### DISCUSSION THREAD

**Discussion by saashqdev** [cloud-ark/kubeplus#1320] [Ideas]
Has anyone tried Supabase with Kubeplus? Sounds like a great way to launch multiple instances in your local environment.
Cheers, Dave

**Comment by devdattakulkarni** [comment]
@saashqdev Yes - we have tried Supabase chart with KubePlus. It works well. 
What is your use-case?

**Comment by saashqdev** [comment]
I just saw it on your test list right after I posted. Should've checked first. Anyway, that's great. The idea was to run multiple instances of Supabase locally for multiple projects. I haven't dove into the project yet, is there a web gui to manage all the instances etc...?

**Reply by devdattakulkarni** [reply]
@saashqdev Yes, there is a basic webui (named consumerui) to manage application instances through UI. This should get you started. If there are any features that you need in the consumerui, let us know. 

In case you are looking for a way to manage supabase instances across multiple clusters then check out our commercial offering from our website. It provides a dashboard to manage application instances across clusters. It also comes with Prometheus and Elastic.

**Comment by saashqdev** [comment]
Great! Thanks for the prompt info. I'll check it out.

**Reply by devdattakulkarni** [reply]
You can join our slack channel on CNCF workspace (#kubeplus) for questions/discussions.
